ui/slider.js

```diff
@@ -564,6 +564,18 @@ return $.widget( "ui.slider", $.ui.mouse, {
 		return this.max;
 	},
 
+	_refreshRange: function ( orientation ) {
+		if ( this.options.range === false ) {
```

Comment:
That's fine, we have tons of checks for whether or not there's a range slider. All of those conditionals will go away eventually anyway since range sliders will become their own widget.